typescript/cli/src/config/warp.ts

```diff
@@ -144,9 +149,11 @@ export async function createWarpRouteDeployConfig({
       'hyperlane-registry',
     );
 
-    const interchainSecurityModule = shouldUseDefault
-      ? await createDefaultWarpIsmConfig(context)
-      : await createIsmConfig(context);
+    const interchainSecurityModule = await createIsmConfig({
+      context,
+      defaultFn: createDefaultWarpIsmConfig,
```

Comment:
I don't know if this is helpful because this callback function is restricted to 1 signature `(context: CommandContext) => Promise<IsmConfig>`. what if i have a function with different args like `(context: CommandContext, modifiedOwner: Address, something: string) => Promise<IsmConfig>`?

Reply:
spoke offline– removed this altogether and moved logs into createAdvancedIsmConfig since we have the advanced specification now